src/redux/events.js

```diff
@@ -0,0 +1,60 @@
+import axios from 'axios';
+const GET_EVENTS = 'GET_EVENTS';
+const CREATE_EVENT = 'CREATE_EVENT';
+const SINGLE_EVENT = 'SINGLE_EVENT'
+
+const initialState = {
+  event: {},
+  events: [],
+};
+
+
+export const _getEvents = (events) => {
+    return {
+      type: GET_EVENTS,
+      events,
+    };
+  };
+
+export const getEvents = () => {
+    try {
```

Comment:
separating out redux store is a lot better than leaving it all in one file 
i usually make another file for a "state"
lets say you have 2 states: 
user, 
events,

file structure would be like 
-user
--thunks.js
--actions.js
--reducer.js
-events
--thunks.js
--actions.js
--reducer.js